src/components/scene/background.js

```diff
@@ -15,6 +16,30 @@ module.exports.Component = register('background', {
       return;
     }
     object3D.background = new THREE.Color(data.color);
+
+    if (data.generateEnvironment) {
+      const scene = this.el.sceneEl.object3D;
+      const cubeRenderTarget = new THREE.WebGLCubeRenderTarget(128, { format: THREE.RGBFormat, generateMipmaps: true, minFilter: THREE.LinearMipmapLinearFilter });
+      const cubeCamera = new THREE.CubeCamera(1, 100000, cubeRenderTarget);
+      this.cubeCamera = cubeCamera;
+      this.needsEnvironmentUpdate = true;
+      scene.environment = cubeRenderTarget.texture;
+      this.el.sceneEl.addEventListener('loaded', function () {
+        this.needsEnvironmentUpdate = true;
+      });
+    }
+  },
+
+  tick () {
+    if (!this.data.auto && !this.needsEnvironmentUpdate) return;
```

Comment:
I don't see `auto` defined in schema.

Reply:
Thanks for pointing that out! That would be an expensive error.